scenarios/multi_object_tracking/01_training_introduction.ipynb

```diff
@@ -0,0 +1,820 @@
+{
```

Comment:
<ul><li>This needs more high-level information, e.g. why are we using this video for evaluation? That's because we can train on still-frame, but evaluation of e.g. ID swtiches needs video</li><li>The whole section should be moved towards the end of the notebook, after model training is done and evaluation starts</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/545/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>